lib/win32.dart

```diff
@@ -263,6 +268,7 @@ export 'src/com/ITypeInfo.dart';
 export 'src/com/IUnknown.dart';
 export 'src/com/IUri.dart';
 export 'src/com/IUserDataPathsStatics.dart';
+export 'src/com/IVector`1.dart';
```

Comment:
I was. Since we were gonna expose methods or properties that return `IVectorView` as a `List` to consumers, I thought it would be a good idea to not export `IVectorView` to consumers. After your question, I remembered that some functions take `IVectorView` as parameters, so I exported it.

Reply:
Yeah, I think it's good to expose it even if we wind up completely wrapping it for convenience.